libs/net/models/Areas/Services/Models/Content/ContentModel.cs

```diff
@@ -176,6 +181,7 @@ public ContentModel(Entities.Content entity) : base(entity)
         this.SourceId = entity.SourceId;
         this.OtherSource = entity.OtherSource;
         this.ProductId = entity.ProductId;
+        this.Product = new ProductModel(entity.Product);
```

Comment:
You need to deal with nullability.
`this.Product = entity.Product != null ? new ProductModel(entity.Product) : null;`

Reply:
Updated, thanks